src/test/java/lotto/CommonValidation/CommonValidationTest.java

```diff
@@ -0,0 +1,102 @@
+package lotto.CommonValidation;
+
+import static lotto.CommonValidation.CommonValidation.hasBlank;
+import static lotto.CommonValidation.CommonValidation.isInteger;
+import static lotto.CommonValidation.CommonValidation.isValidLottoNumber;
+import static lotto.Dictionary.LottoDictionary.LOTTO_MAX_NUMBER;
+import static lotto.Dictionary.LottoDictionary.LOTTO_MIN_NUMBER;
+import static lotto.Message.ExceptionMessage.CommonValidationErrorMessage.*;
+import static org.assertj.core.api.Assertions.*;
+
+import lotto.Exception.CommonValidationException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class CommonValidationTest {
+
+    private final int invalidLottoNumber1 = 0;
+    private final int invalidLottoNumber2 = 46;
+    private final String whiteSpace = " ";
+    private final String notInteger = "a";
+    private final String realNumber = "1.1";
+    private final String normalValue = "1";
+
+    @Test
+    @DisplayName("숫자가 범위 밖에 있을때 예외가 발생하는지 확인하는 테스트")
```

Comment:
테스트의 이름은 "~ 상황일 때 ~ 하면 ~ 되어야 한다." 식으로 상황을 평문으로 서술해 주는 것이 가장 명확하게 테스트를 설명할 수 있다고 생각합니다.
```suggestion
    @DisplayName("범위 밖 숫자를 이용해 isValidLottoNumber()를 수행하면 예외가 발생해야 한다.")
```

Reply:
테스트 명에 대해서 좋은 의견 주셔서 감사합니다!